postgres-appliance/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+DOCKERCMD="docker build"
+
+function usage()
+{
+    cat <<__EOF__
+Usage: $0 [DOCKER ARGUMENTS]
+
+__EOF__
+}
+
+REV=$(git rev-parse HEAD)
+URL=$(git config --get remote.origin.url)
+STATUS=$(git status --porcelain)
+GITAUTHOR=$(git show -s --format="%aN <%aE>" "$REV")
+
+cat > scm-source.json <<__EOT__
+{
+    "url": "$URL",
+    "revision": "$REV",
+    "author": "$GITAUTHOR",
+    "status": "'$STATUS'"
```

Comment:
@feikesteenbergen no, the docs are right (single quotes are needed to end the outer string and to interpret the env var)